packages/next-auth/src/providers/instagram.js

```diff
@@ -35,6 +35,9 @@ export default function Instagram(options) {
     token: "https://api.instagram.com/oauth/access_token",
     userinfo:
       "https://graph.instagram.com/me?fields=id,username,account_type,name",
+    client: {
+      token_endpoint_auth_method: 'client_secret_post',
```

Comment:
Thanks! That was my assumption as well in https://github.com/nextauthjs/next-auth/pull/3381#discussion_r775294734